spacelift/resource_module_test.go

```diff
@@ -114,4 +115,30 @@ func TestModuleResource(t *testing.T) {
 			},
 		})
 	})
+
+	for _, name := range []string{
+		"github-Module",
+		"github-module-",
+		"_github-module",
+		"0github-module",
+	} {
+		t.Run("invalid name", func(t *testing.T) {
+			config := func(description string, protectFromDeletion bool) string {
```

Comment:
nit: I don't think `description` or `protectFromDeletion` are used anymore, so you could just inline this.

Reply:
Good point, fixed.